embedded/servo_driver/servo_test.cc

```diff
@@ -0,0 +1,19 @@
+#include "servo_driver.hh"
+
+ServoDriver driver = ServoDriver("/dev/i2c-1");
```

Comment:
[nit] Avoid globals, prefer to initialize everything inside of a function; here it would be `main`